imagespace/server/imagesearch_rest.py

```diff
@@ -101,16 +102,23 @@ def filenameUpper(filename):
             query += ' AND (%s)' % ' OR '.join(['%s:[.7 TO *]' % key
                                                 for key in classifications])
 
+        qparams = {
+            'wt': 'json',
+            'hl': 'true',
+            'hl.fl': '*',
+            'q': query,
+            'start': offset,
+            'rows': limit
+        }
+
+        # Give plugins a chance to adjust the Solr query parameters
+        event = events.trigger('imagespace.imagesearch.qparams', qparams)
```

Comment:
What does having `qparams` as an argument get you in this call? Seems the plugin doesn't directly edit this so why is it needed on trigger?